subsys/mpsl/fem/nrf2240/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+zephyr_library_sources(mpsl_fem_nrf2240.c)
```

Comment:
Isn't `nrfx_twim_stub.c` needed here as well, exactly as for nRF2220?

Reply:
Please have a look on introduced now Kconfig `MPSL_FEM_USE_TWIM_STUB`. This option turns on the build of `nrfx_twim_stub.c` which is moved to `subsys/mpsl/fem/common/`

Reply:
Perfect